### DISCUSSION TITLE
Marking a collection as "system" in Admin UI

### DISCUSSION THREAD

**Discussion by mrbu-solutions** [pocketbase/pocketbase#2840] [Q&A] [answered]
Hi Pocketbase Team,

How can I mark a collection as system in the Admin UI?
I tried to look at the Settings Menu, and at the Collection settings, but I found none.

However, in the documentation, it is made possible via Web API.

![image](https://github.com/pocketbase/pocketbase/assets/53817867/f0dc072e-3f0d-424e-bf3a-2da0da2fab8a)

Do you consider providing it via Admin UI in the future?

Use case is I want to mark some collections as system like ApplicationSettings, Configuration, etc... collections so I won't accidentally delete it or other admins.

Thank you very much.

**Comment by ganigeorgiev** [comment, marked as answer]
I may consider it in the future, but the main reason to not have the option in the UI is that once enabled it cannot be removed easily and it is preferred usually to be used in a migration script instead.
_The initial purpose of the `system` option was because of the `profiles` collection before the [users-profiles merge](https://github.com/pocketbase/pocketbase/issues/376) < v0.8._

**Reply by mrbu-solutions** [reply]
Thank you very much!
